boards/components/src/debug_writer.rs

```diff
@@ -31,7 +31,7 @@ use kernel::hil::uart;
 // least a 1 KiB boundary). This is not _semantically_ critical, but helps keep buffers on 1 KiB
 // boundaries in some cases. Of course, these definitions are only advisory, and individual boards
 // can choose to pass in their own buffers with different lengths.
-pub const DEBUG_BUFFER_KBYTE: usize = 1;
+pub const DEBUG_BUFFER_KBYTE: usize = 8;
```

Comment:
Hopefully this was just for debugging? This is a big increase

Reply:
I think it's a good idea to have such an increase with this particular driver. It allows userspace to be able to dump a lot of information. 1K is often too small, for example, for what one of our kernel dumps looks like. I suppose the issue is that this is shared across platforms, and a RAM-starved one might not want this?

Also worth discussing. I rarely find RAM tight on systems, but I do often find debug overflows when debugging. :)

Reply:
I think this change would break the HiFive board today. We should probably just adjust the component to add an optional paramater to the macro, so the HiFive can maintain a smaller debug buffer than other boards.

Reply:
Submitted https://github.com/tock/tock/pull/3369 to address this